### ISSUE TITLE
convert dict object to `$key : $value`

### ISSUE THREAD

**Issue by knknkn1162** [knknkn1162/mdrec#3]
MDRec.rec allows dict to convert 

```code
`$key : $value`
```

or

```code
<dl>
  <dt>key</dt>
  <dd>value</dd>
</dl>
```

**Comment by knknkn1162** [comment]
component.enum allows dict to convert only the form, `$key : $value` with #11

**Comment by knknkn1162** [comment]
fix #14

**Comment by knknkn1162** [comment]
No support latter part. 

```html
<dl>
  <dt>key</dt>
  <dd>value</dd>
</dl>
```
